### DISCUSSION TITLE
Add new decorator to document an endpoints requestBody without necessarily validating it, like @app.input does

### DISCUSSION THREAD

**Discussion by IodizedSalt** [apiflask/apiflask#493] [General]
Perhaps this exists already, but from my reading of documentation, it doesn't seem so.

I simply wish to keep the current functionality of @app.input where I can define a schema to be displayed in the docs for the required requestBody. However, my API is using custom responses/error handling, and I do not wish to use @app.input built in validation/error handling.

Is there some way to solely use the documentation component of this decorator, without the built in validation ? 

Edit:
Hmm, I took another glance at the documentation and completely missed [the error handling section's part about overriding ValidationError](https://apiflask.com/error-handling/)

I reckon I can just update the ValidationError body directly here to fit my needs, but I still would have liked an extra decorator for simply documenting the requestBody without validation, but perhaps it's bloat. :)

**Comment by greyli** [comment]


**Reply by IodizedSalt** [reply]
That's almost perfect, but I need something more for the request.data/payload to the endpoint, so in addition to `responses`, maybe a `requests` or `requestBody` parameter?

Otherwise there might be something else existing I havn't found?
